pyutilib/misc/tests/test_import.py

```diff
@@ -83,6 +98,14 @@ def test_import_file_context3(self):
         if not "import1" in globals():
             self.fail("test_import_file - failed to import the import1.py file")
 
+    def test_import_exception(self):
+        orig_path = list(sys.path)
+        with self.assertRaisesRegexp(RuntimeError, "raised during import"):
+            pyutilib.misc.run_file(
```

Comment:
Should this be calling `pyutilib.misc.import_file` instead of `pytuilib.misc.run_file`? The test is on the `TestImportFile` class.